statuscake/model_utils.py

```diff
@@ -54,18 +55,18 @@ def convert_js_args_to_python_args(fn):
     from functools import wraps
 
     @wraps(fn)
-    def wrapped_init(_self, *args, **kwargs):
+    def wrapped_init(__self, *args, **kwargs):
         """
-        An attribute named `self` received from the api will conflicts with the reserved `self`
-        parameter of a class method. During generation, `self` attributes are mapped
-        to `_self` in models. Here, we name `_self` instead of `self` to avoid conflicts.
+        An attribute named `self` received from the api will conflicts with the
```

Comment:
will conflict

Reply:
Yes this is what the comment is explaining. The change it talks about avoids the conflict.